api/cli/c2cx.go

```diff
@@ -25,19 +26,29 @@ func sumbitTradeCMD() cli.Command {
 				return errInvalidInput
 			}
 			var (
-				symbol        string
-				price, amount float64
+				symbol                             string
+				price, amount                      decimal.Decimal
+				stopLoss, takeProfit, triggerPrice decimal.Decimal
 			)
+			if stopLoss, err := decimal.NewFromString(stoploss); err != nil {
```

Comment:
the `stoploss`, `stopLoss` etc variables look mishandled here?

Reply:
I thought the best way to get the most accurate `decimal.Decimal` values was to parse them as string flags, and then pass the strings to `decimal.NewFromString`.

`stoploss` is the raw string parameter and `stopLoss` is the parsed `decimal.Decimal`.

I just [updated the code to be more explicit with the variable names](https://github.com/skycoin/exchange-api/pull/31/commits/d05e698b7139e5eb2f69d5825852c5a6f92ec825).

(Or should I do it a different way?)